### ISSUE TITLE
Investigate routing outage

### ISSUE THREAD

**Issue by ddohler** [azavea/echo-locator#495]
Seems to be affecting all of development, staging, and production.

**Comment by ktohalloran** [comment]
The issue comes down to the fact that the Cloudfront URLs that are used for routing are no longer active. It has impacted routing across all versions of the app, including the demo site. In investigating this issue, I answered two questions: 1) does the domain of these Cloudfront URLs belong to us and therefore requires a fix on our end? and 2) If not, what is the issue with those URLs?

In answer to the first question, it is clear that we do not own the domain, as reflected in Katherine's exchange about the bucket URLs in issue #140. I also verified that the domain used by the routing is not one of the four listed in our own Cloudfront for this project.

In answer to the second, the issue is that the Couldfront domain that these URLs relied on has either been taken down or moved. I verified this by sending a request to https://d2z7d5345ccuw9.cloudfront.net via Postman, which resulted in a `getaddrinfo ENOTFOUND` error; similarly, running `dig https://d2z7d5345ccuw9.cloudfront.net` revealed the URL was no longer connected with any server at all. As a result, it's clear that this issue is unfortunately not something we can fix ourselves.
